src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -110,7 +110,7 @@ const DetailDomainScreenCommentItem = ({
         <TouchableOpacity
           style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}
           onPress={() => onBlock(comment)}>
-          <IconEn name="block" size={15.02} color={COLORS.lightgrey} />
+          <IconEn name="block" size={15.02} color={COLORS.gray100} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the block icon has been changed from `COLORS.lightgrey` to `COLORS.gray100`. Ensure that this color change aligns with the overall design and accessibility guidelines, as it could impact visibility and user experience.
```diff
- <IconEn name="block" size={15.02} color={COLORS.lightgrey} />
+ <IconEn name="block" size={15.02} color={COLORS.gray100} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->